extensions/f1-standings/src/useSeasons.ts

```diff
@@ -0,0 +1,34 @@
+import { useState, useEffect } from "react";
+import fetch from "node-fetch";
+import { popToRoot, showToast, Toast } from "@raycast/api";
+
+interface Season {
+  season: number;
+  url: string;
+}
+
+export const useSeasons = () => {
+  const [seasons, setSeasons] = useState<Season[]>([]);
+
+  useEffect(() => {
+    async function fetchDrivers() {
```

Comment:
Maybe this should be called "fetchSeasons"

Reply:
yup!